client/src/components/ui/search.tsx

```diff
@@ -0,0 +1,30 @@
+import { Search as SearchIcon } from 'lucide-react';
+import { useDebounce } from 'rooks';
+
+import { Input } from '@/components/ui/input';
+type SearchProps = {
+  className?: string;
+  onChange: (newValue: string) => void;
+};
+
+const Search = ({ className, onChange }: SearchProps) => {
+  const debouncedOnChange = useDebounce((newValue: string) => {
+    onChange(newValue);
+  }, 250);
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    debouncedOnChange(event.target.value);
+  };
+
+  return (
+    <Input
```

Comment:
Please add an `aria-label` to this component. You can give it the same value as the placeholder.